support/mock-data/Terms/expand/Genetics/HealthProfessional/es/Y.json

```diff
@@ -0,0 +1,8 @@
+{
+    "meta": {
+      "totalResults": 0,
+      "from": 0
+    },
+    "results": [],
+    "links": null
+  }
```

Comment:
if you run format command on these files, it will add the newline for you

Reply:
Formated all!